sources/datadog.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: datadog
+  namespace: flux-system
+spec:
+  interval: 5m0s
+  url: https://helm.datadoghq.com
```

Comment:
Don't forget to configure the Helm exporter for this chart:
https://github.com/dfds/eks-pipeline/blob/master/services.hcl#L185-L219

Reply:
Thanks, I wouldn't have thought of that 😓